src/runners/utils.ts

```diff
@@ -50,13 +50,16 @@ export function createDefaultServer(
                 await dependencies.connection.close();
             }
             if (!server.listening) {
+                logger.flush();
```

Comment:
It looks like all these codepaths lead to flush at the end. Is there a reason to do multiple flushes?

Reply:
You can see the `logger.flush()` precedes `process.exit`, we can make it a bit nice by lifting it to a function:
```
const exitWithCode = (exitCode) => {
                logger.flush();
                process.exit(exitCode);

};
```